generate-extension-zip.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Creates a zip of (only) the necessary files required by Gnome for the PaperWM extension.
+# Designed for submitting a zip to extensions.gnome.org.
+/usr/bin/zip -r paperwm@paperwm.github.com.zip \
```

Comment:
```suggestion
zip -r paperwm@paperwm.github.com.zip \
```

Why are you using an absolute path?